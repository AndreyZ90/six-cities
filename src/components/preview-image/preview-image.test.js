import React from 'react';
import TestRenderer from 'react-test-renderer';

import PreviewImage from '@/components/preview-image/preview-image';

const src = `img/image.jpg`;

describe(`PreviewImage component snapshot`, () => {
  test(`Should correctly render PreviewImage component`, () => {
    const tree = TestRenderer.create(<PreviewImage src={src} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
