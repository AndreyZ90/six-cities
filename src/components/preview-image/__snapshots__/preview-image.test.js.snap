// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreviewImage component snapshot Should correctly render PreviewImage component 1`] = `
<div
  className="place-card__image-wrapper"
>
  <a>
    <img
      alt="Place image"
      className="place-card__image"
      height={200}
      src="img/image.jpg"
      width={260}
    />
  </a>
</div>
`;
